charts/cla/templates/service.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "cla.fullname" . }}
+  labels:
+    {{- include "cla.labels" . | nindent 4 }}
+spec:
+  type: {{ .Values.service.type }}
+  ports:
+    - name: cla
+      port: {{ .Values.service.port }}
+      nodePort: {{ .Values.service.nodeport }}
```

Comment:
Why do you need to specify this? Also, it only makes sense when the service is of type `NodePort`, otherwise, it will be ignored